addons/grenades/XEH_postInit.sqf

```diff
@@ -10,6 +10,7 @@
 ["ace_firedNonPlayer", DFUNC(throwGrenade)] call CBA_fnc_addEventHandler;
 
 if (!hasInterface) exitWith {};
+call FUNC(changeFuse);
```

Comment:
```suggestion
[] call FUNC(changeFuse);
```

I think the function name is not descriptive enough, `addChangeFuseItemContextMenuOptions` ?